### ISSUE TITLE
Running from file in same directory requires leading `./`

### ISSUE THREAD

**Issue by martinklepsch** [retrogradeorbit/bootleg#61]
Somehow this works:
```
bootleg -e '(doseq [f (glob "content/posts/*.markdown")] (println f))'
```
but this fails on my machine (MacOS 10.14.6), bootleg `Version: 0.1.7`:
```
cat '(doseq [f (glob "content/posts/*.markdown")] (println f))' >> bootlegrepro.clj
bootleg bootlegrepro.clj
```
```
Exception in thread "main" java.lang.NullPointerException
	at bootleg.file$relativise.invokeStatic(file.clj:57)
	at bootleg.file$relativise.invoke(file.clj:53)
	at clojure.core$partial$fn__5824.invoke(core.clj:2624)
	at clojure.core$map$fn__5851.invoke(core.clj:2755)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.RT.seq(RT.java:531)
	at clojure.core$seq__5387.invokeStatic(core.clj:137)
	at clojure.core$seq__5387.invoke(core.clj:137)
	at sci.impl.interpreter$fn_call.invokeStatic(interpreter.cljc:428)
	at sci.impl.interpreter$fn_call.invoke(interpreter.cljc:428)
	at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:474)
	at sci.impl.interpreter$eval_call.invoke(interpreter.cljc:460)
	at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:503)
	at sci.impl.interpreter$interpret.invoke(interpreter.cljc:492)
	at sci.impl.interpreter$fn_call.invokeStatic(interpreter.cljc:428)
	at sci.impl.interpreter$fn_call.invoke(interpreter.cljc:428)
	at sci.impl.interpreter$eval_call.invokeStatic(interpreter.cljc:474)
	at sci.impl.interpreter$eval_call.invoke(interpreter.cljc:460)
	at sci.impl.interpreter$interpret.invokeStatic(interpreter.cljc:503)
	at sci.impl.interpreter$interpret.invoke(interpreter.cljc:492)
	at sci.impl.interpreter$eval_form.invokeStatic(interpreter.cljc:547)
	at sci.impl.interpreter$eval_form.invoke(interpreter.cljc:538)
	at sci.impl.interpreter$eval_string_STAR_.invokeStatic(interpreter.cljc:557)
	at sci.impl.interpreter$eval_string_STAR_.invoke(interpreter.cljc:550)
	at sci.impl.interpreter$eval_string$fn__1514.invoke(interpreter.cljc:570)
	at sci.impl.interpreter$eval_string.invokeStatic(interpreter.cljc:567)
	at sci.impl.interpreter$eval_string.invoke(interpreter.cljc:563)
	at sci.core$eval_string.invokeStatic(core.cljc:169)
	at sci.core$eval_string.invoke(core.cljc:137)
	at bootleg.hiccup$process_hiccup_data.invokeStatic(hiccup.clj:90)
	at bootleg.hiccup$process_hiccup_data.invoke(hiccup.clj:22)
	at bootleg.hiccup$process_hiccup.invokeStatic(hiccup.clj:106)
	at bootleg.hiccup$process_hiccup.invoke(hiccup.clj:97)
	at bootleg.core$process.invokeStatic(core.clj:37)
	at bootleg.core$process.invoke(core.clj:35)
	at bootleg.core$_main.invokeStatic(core.clj:77)
	at bootleg.core$_main.doInvoke(core.clj:57)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at bootleg.core.main(Unknown Source)
```

**Comment by retrogradeorbit** [comment]
what is inside "content/posts/"? Something is failing in trying to turn paths to relative. Might be a bug. Can you share the directory contents of that folder?

**Comment by martinklepsch** [comment]
I actually figured it out just now: `bootleg ./bootlegrepro.clj` just works! Note the leading `./` before the filename to be executed.

**Comment by retrogradeorbit** [comment]
ahh good find

**Comment by retrogradeorbit** [comment]
Fixed in v0.1.8
